hardware/arduino/avr/cores/arduino/wiring.c

```diff
@@ -38,6 +38,9 @@
 volatile unsigned long timer0_overflow_count = 0;
 volatile unsigned long timer0_millis = 0;
 static unsigned char timer0_fract = 0;
+static (*TIM0_Callback)(void) = NULL;
+
+#define CALLBACKDIVIDER		16
```

Comment:
The style convention established in this file and through much of the rest of the core is for the macro name to be separated from the definition by a single space.